code/modules/events/infestation.dm

```diff
@@ -23,7 +22,7 @@
 
 /datum/event/infestation/start()
 
-	location = rand(0,9)
+	location = rand(0,8)
```

Comment:
Not a priority but this should really be something like rand(LOC_KITCHEN,LOC_TACTICAL) or something.